prov/shm/src/smr.h

```diff
@@ -184,6 +185,9 @@ static inline const char *smr_no_prefix(const char *addr)
 #define smr_fast_rma_enabled(mode, order) (!smr_env.disable_cma && \
 			(mode & FI_MR_VIRT_ADDR) && \
 			!(order & SMR_RMA_ORDER))
+#define smr_addr_offset(end, start) (uintptr_t)((char *) end - \
+				    (char *) start)
+#define smr_addr_end(start, offset) ((char *) start + (uintptr_t) offset)
```

Comment:
Can we rename these parameter names and the second function name? It isn't clear to me with those names which parameter is which, specifically what "end" means.
Maybe
```
smr_get_offset(base, addr)
smr_get_addr(base, offset)
```

Reply:
Done.